core-web/libs/edit-content/src/lib/components/dot-edit-content-form/dot-edit-content-form.component.scss

```diff
@@ -37,9 +37,23 @@ $tab-min-height: 52px;
 // Edit Content Actions
 .dot-edit-content-actions {
     display: flex;
-    flex-wrap: wrap;
+    align-items: center;
+    width: 100%;
+    min-height: $tab-min-height;
+}
+
+.dot-edit-content-actions__left {
+    display: flex;
     align-items: center;
     gap: $spacing-4;
+    min-width: 200px; // Adjust this value based on your needs
+}
+
+.dot-edit-content-actions__right {
+    display: flex;
```

Comment:
I think we should use more prime flex utilities instead of adding more css classes, in this case:

```css
<div class="flex align-items-center gap-4 ml-auto">
```